nfts/frontend/src/utils/transferNft.js

```diff
@@ -0,0 +1,37 @@
+import fetch from "node-fetch";
+export default async function getNftTransferConsent(
+  tokenId,
+  toEmail,
+  fromEmail
+) {
+  console.log("Transferring NFT...");
+  const url = "http://localhost:3001/getConsentToken";
+  const headers = {
+    "Content-Type": "application/json",
+    Accept: "application/json",
```

Comment:
Also add quotes to Accept

Reply:
done

Reply:
not done

Reply:
prettier changes it back to without quotes